### PR TITLE
created a django rest api which return information of trips

### PR DESCRIPTION
**Pull request by Muazam-Husnain**
Created a django rest api which returns information of all trip on url 'api/v1/trips'
Screenshot 2020-10-15 at 3 48 31 PM

### CODE REVIEW

File: learn_django/trip_project/serializers.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+from .models import Host, Location, Trip, Itenrary
+
+class TripSerializer(serializers.ModelSerializer):
+    host = serializers.StringRelatedField()
```
Comment (awaisdar001) [inline learn_django/trip_project/serializers.py]: Why `StringRelatedField` is needed for `host`?

**Review by awaisdar001 — CHANGES_REQUESTED** [review]
Have separate serializers for host and destination_location.

**Comment by Muazam-Husnain** [comment]
Screenshot 2020-10-27 at 12 39 10 PM

**Review by awaisdar001 — CHANGES_REQUESTED** [review]
I need data as 
```
...
"host": {"name": "Travel guide of Pakistan", "number" : 123},
"destination_location": {"name": "Lahore"},
"start_location": {"name": "Lahore"},
```


2. I would not expose `id` field in the API. 

3. I would like to add 10 trips and then share a screenshot.

**Comment by Muazam-Husnain** [comment]
Screenshot 2020-10-27 at 2 15 10 PM
Screenshot 2020-10-27 at 2 15 06 PM

**Review by awaisdar001 — APPROVED** [review]
This looks good. 

* You need to know how to rename field names in APIs without changing the database table attributes. 
* Do not expose `id` anywhere in the models. the right solution here was to use `exclude` rather than manually typing in all the field names in `include`. 


have those changes and you can merge this PR.

**Pull request merged**